src/MotionControl.c

```diff
@@ -1436,6 +1430,34 @@ BOOL Ros_MotionControl_StartMotionMode(MOTION_MODE mode)
         }
     }
 
+    // Check if in continous cycle mode
+    if (!Ros_Controller_IsContinuousCycle())
+    {
+        // set the cycle mode to auto if not currently
+        MP_CYCLE_SEND_DATA sCycleData;
+        bzero(&sCycleData, sizeof(sCycleData));
+        bzero(&rData, sizeof(rData));
+        sCycleData.sCycle = 3;  // Auto
+        ret = mpSetCycle(&sCycleData, &rData);
+        if( (ret != 0) || (rData.err_no != 0) ) 
+        {
+            Ros_Debug_BroadcastMsg(
+                "Can't set cycle mode to continuous because: '%s' (0x%04X)",
+                Ros_ErrorHandling_ErrNo_ToString(rData.err_no), rData.err_no);
+            mpSetAlarm(ALARM_OPERATION_FAIL, "Set job-cycle to AUTO", SUBCODE_OPERATION_SET_CYCLE);
```

Comment:
This message doesn't appear to correspond to what the documentation shows?

(same below)

Reply:
Oops. I changed my mind on the alarm msg after writing the doc. It is now updated.